src/components/Comments/ViewComments/ViewComments.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import cookie from 'react-cookies';
+import {
+  Typography, Avatar, List, ListItem, ListItemAvatar, ListItemText,
+} from '@material-ui/core';
+import useStyles from './style';
+import { ErrorMessage } from '../../ErrorMessage';
+import { serviceRequest } from '../../../services/serviceRequest';
+import { useStateValue } from '../../StateProvider';
+import { Pagination } from '../../Pagination';
+// import config from '../../../config';
+
+// const path = '/api/v1/post/viewcomments';
+// const domain = config.apiDomain;
+
+const preparePayload = (method, headers, params) => {
+//   const url = `https://${domain}${path}`;
+  const url = 'http://localhost:8081/api/v1/post/viewcomments';
```

Comment:
I am curious why u use local api to testing

Reply:
the code was before the apis are live. I have changed it to live api locally, didn't push yet